recipe-server/normandy/health/tests/test_api.py

```diff
@@ -18,6 +25,7 @@ def test_version(client, mocker):
         'commit_link': 'https://github.com/mozilla/normandy/commit/<git hash>',
         'configuration': 'Test',
         'version': '<tag>',
+        'build_time': Whatever(),
```

Comment:
We could also just save the datetime above and compare it to `dt.isoformat()` here.